### DISCUSSION TITLE
Tab navigator in NextJS with Expo Router

### DISCUSSION THREAD

**Discussion by META-DREAMER** [nandorojo/solito#218] [Q&A]
Is there any example of how one could achieve a bottom tab navigator in both NextJS and native, where it switches to a side-tab layout on desktop (i.e. same UX as Twitter mobile > desktop)

![image](https://user-images.githubusercontent.com/7143583/197273637-bf9acd34-2304-4c8e-8417-f1f58310984e.png)
![image](https://user-images.githubusercontent.com/7143583/197273736-f022fe10-31d2-44d9-ad47-d7ebe011562e.png)

**Comment by TakshPanchal** [comment]
I guess it is responsive UI logic, isn't it?

**Comment by nandorojo** [comment]
The we styles should be totally separate from the native tabs for the navigation layout.

**Reply by poonamm-cemtrexlabs** [reply]
means??
